gk/main.c

```diff
@@ -1817,6 +1836,8 @@ process_pkts_front(uint16_t port_front, uint16_t port_back,
 			flow_hash_val_arr[fidx], fibs[i], &num_tx, tx_bufs,
 			acl4, acl6, num_pkts, icmp_bufs, req_bufs, req_prio,
 			&num_reqs, front, back, instance, gk_conf);
+		if (fe_arr[fidx])
+			prefetch_flow_entry(fe_arr[fidx]);
```

Comment:
The only ocasion `lookup_fe_from_lpm()` returns a flow entry is when the corresponding fib is at state `GK_FWD_GRANTOR`. In this state, `initialize_flow_entry()` writes the flow entry without reading anything from the content of the flow entry. Therefore, this prefetch and its IPv6 corresponding below shouldn't be helping. Could you drop them and evaluate the load overhead with perf?

Reply:
Sure, will do.

Reply:
Indeed, dropping these lines of code won't affect the load overhead.